src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { useAppContext } from '../context/AppContext';
+import { User } from '../User';
+
+export const PeopleList = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isErrorToGetPeople, setIsErrorToGetPeople] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { hasClickedPeopleLink } = useAppContext();
+  const { slug } = useParams();
+  const selectId = slug || '';
+  const selectedUser = people.find(p => p.slug === selectId);
+
+  useEffect(() => {
+    if (hasClickedPeopleLink) {
+      getPeople()
+        .then((allPersons) => setPeople(allPersons))
+        .catch(() => {
+          setIsErrorToGetPeople(true);
+          throw new Error(' Something went wrong');
+        })
+        .finally(() => setIsLoading(false));
+    }
+  }, [hasClickedPeopleLink]);
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        <h1 className="title">People Page</h1>
+
+        {isErrorToGetPeople && (
+          <p data-cy="peopleLoadingError" className="has-text-danger">
+            Something went wrong
+          </p>
+        )}
+
+        {(!people.length && !isErrorToGetPeople) && (
+          <Loader />
+        )}
+
+        {!!people.length && (
+          <table
+            data-cy="peopleTable"
+            className="table is-striped is-hoverable is-narrow is-fullwidth"
+          >
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Sex</th>
+                <th>Born</th>
+                <th>Died</th>
+                <th>Mother</th>
+                <th>Father</th>
+              </tr>
+            </thead>
```

Comment:
You may consider creating Enum with this values and then unroll them with .map into column titles, just as we did with 'todos' filters.

Reply:
in my opinion its not good idea

Reply:
Not fixed